demos/node-classification/node2vec/stellargraph-node2vec-weighted-random-walks.ipynb

```diff
@@ -555,16 +575,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 18,
+   "execution_count": 19,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "0.792604501607717"
+       "0.4340836012861736"
```

Comment:
The accuracy here and on lines 805 and 1122 has dropped massively? Has something gone wrong here, or is it just very variable?

Reply:
well spotted!  I'm not sure what happened here, however I re-ran and got much better results (now pushed); it's possible that I accidentally committed results from a run with altered parameters.

Reply:
Ah, right, I guess `walk_length = 5` would reduce performance...